jac-uk/apply
# remove static vacancies

## What's included?
Remove static [off platform] old JARs era exercise links and supporting logic. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Visit vacancies page
- ensure below exercises DO NOT appear in the list 
- 
![image](https://user-images.githubusercontent.com/44227249/169562434-2054767c-2c7f-46ed-a051-54459f96e92d.png)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
